tests/testsoon.nim

```diff
@@ -46,9 +46,11 @@ suite "callSoon() tests suite":
       await sleepAsync(100.milliseconds)
       timeoutsTest1 += 1
 
-  proc callbackProc(udata: pointer) {.gcsafe.} =
+  var callbackproc: proc(udata: pointer) {.gcsafe, raises: [Defect].}
+  callbackproc = proc (udata: pointer) {.gcsafe, raises: [Defect].} =
     timeoutsTest2 += 1
-    callSoon(callbackProc)
+    {.gcsafe.}:
```

Comment:
Why is this override needed here?

Reply:
probably it isn't with `unittest2`, but before, callbackproc was a global - I'll see if it can be removed